website/docs/docs/mesh/iceberg/bigquery-iceberg-support.md

```diff
@@ -58,9 +60,11 @@ Supply and nest these additional configurations, unique to BigQuery, under the `
 | `base_location_subpath` | String | No     | An optional suffix to add to the `base_location` path that dbt automatically specifies.     | Only configurable per-model |
 | `storage_uri` | String | No     | If provided, the input will override the dbt storage_uri value. | Only configurable per-model |
 
-These properties can be set in model configurations under the `adapter_properties` field, or as top-level fields themselves. If present in both places, the value set under `adapter_properties` will take precedence.
+These properties can be set in model configurations under the `adapter_properties` field, or as top-level fields themselves. If present in both places, the value set under `adapter_properties` will take precedence. Refer to [Base location](#base-location) for more information.
 
-See [Base location](#base-location) for more information.
+- **base_location_root:** Specifies the prefix of the base location path within the storage bucket where Iceberg table data will be written.
+- **base_location_subpath:** Specifies the suffix of the base location path within the storage bucket where Iceberg table data will be written. This property can only be set in model configurations, not in `catalogs.yml`.
+- **storage_uri:** Completely overrides the storage_uri, allowing you to specify the full path directly instead of using the catalog integration's external volume and base_location components.
```

Comment:
```suggestion
- `storage_uri`: Completely overrides the storage_uri, allowing you to specify the full path directly instead of using the catalog integration's external volume and base_location components.
```